Slack.Automation/Promact.Core.Repository/TaskMailRepository/TaskMailRepository.cs

```diff
@@ -173,7 +173,7 @@ public TaskMailRepository(IRepository<TaskMail> taskMail, IStringConstantReposit
                                         // if previous question was hour of task and it was not null/wrong value then answer will ask next question
                                         var hour = Convert.ToDecimal(answer);
                                         // checking range of hours
-                                        if (hour > 0 && hour < 8)
+                                        if (hour > 0 && hour <= 8)
```

Comment:
Naming convention are improved in issue #105 Stream line code of Task Mail Module
Ronak Shah Already working on it.
Logical implementation is optimal as per me of my part. It would be better if you could suggested me the error.
Wrong usage of framework API, I didn't get that.